scripts/showinfo.js

```diff
@@ -0,0 +1,13 @@
+function deploymenu() {
+  const menubar = document.getElementById('menubar');
+  menubar.style.display = 'flex';
+}
+document.getElementById('deploy-menu').addEventListener('click', deploymenu);
+
+function closemenu() {
+  const menubar = document.getElementById('menubar');
+  menubar.style.display = 'none';
+}
+document.getElementById('closebutton').addEventListener('click', closemenu);
+
```

Comment:
**[OPTIONAL]**

- Please, let's improve our naming convention to make our code more readable. As of right now, your function name and variable are not following the `camelCase` naming convention used in JavaScript, as this is considered bad practice. please let's try and improve this to make our code readable and maintainable.


e.g.. `deployMenu`, `closeMenu`, `menuBar`

Reply:
ok I change every function and every variable into camelCase